### PR TITLE
zsh: amend completion for networkctl edit

### PR DESCRIPTION
**Pull request by rpigott**
Also remove the use of awk here, as it is unnecessary.

### CODE REVIEW

**Review by yuwata — COMMENTED** [review]
Thanks. Could you also add `cat`, `--no-reload`, and `--drop-in` ?

**Comment by rpigott** [comment]
Ok, is that better?

File: shell-completion/zsh/_networkctl
Comment (yuwata) [inline shell-completion/zsh/_networkctl]: Please adjust the indentation.

Also, `_links` is duplicated?

Comment (rpigott) [reply shell-completion/zsh/_networkctl]: > Please adjust the indentation.

Whoops, will do.

> Also, _links is duplicated?

That's intentional. Even though they are flags, _describe only takes completion options in that final argument position so I need something to pass in the 3rd argument position. Passing _links twice allows that without changing the completions.

**Comment by YHNdnzj** [comment]
Thank you!

**Review by yuwata — APPROVED** [review]
Superficially LGTM.
But I am not familiar with completion for zsh, and I do not use zsh.
Let's wait for another review for a while.

**Comment by bluca** [comment]
note sure anybody else here uses zsh

**Pull request merged**
